pipeline/src/client.py

```diff
@@ -135,13 +149,23 @@ def process_data(self, start_date=None, end_date=None, restart=False):
 
         flagged_rows = []
         skipped_rows = 0
+
+        csv_service_keys = []
+        # TODO: Stackoverflow is telling me iterrows is a slow way of iterrating,
+        # but i'll leave optimizing for later.
```

Comment:
What is this comment?

Reply:
This is not mine, from dev I believe

Reply:
It's not in dev. It's added from your changes.